Sources/Screens/RepositoryDetail/RepositoryDetailView.swift

```diff
@@ -1,11 +1,111 @@
 import SwiftUI
 
 struct RepositoryDetailView: View {
-    @StateObject
+    @ObservedObject
     var viewModel: ViewModel
+    var actionHandler: ((Destination) -> Void)?
 
     var body: some View {
-        Text("\(viewModel.repository.name)")
+        NavigationView {
+            List {
+                Section {
+                    HStack {
+                        Text("\(viewModel.repository.name)")
+                            .textFontModifier(size: 20, weight: .bold)
+                        Spacer()
+                        Image(systemName: "greaterthan")
+                    }
+                    .contentShape(Rectangle())
+                }
+                .onTapGesture {
+                    if let url = URL(string: viewModel.repository.htmlUrl) {
+                        UIApplication.shared.open(url)
+                    }
+                }
+                Section("COUNTS") {
+                    HStack(alignment: .firstTextBaseline) {
+                        Text("⭐️")
+                            .countsTextModifier()
+                        Text("stars".capitalized)
+                        Spacer()
+                        Text("\(viewModel.repository.stargazersCount)")
+                    }
+                    HStack {
+                        Text("⫚")
+                            .countsTextModifier()
+                        Text("Forks")
+                        Spacer()
+                        Text("\(viewModel.repository.forks)")
+                    }
+                    HStack {
+                        Text("👀")
+                            .countsTextModifier()
+                        Text("Watchers")
+                        Spacer()
+                        Text("\(viewModel.repository.watchers)")
+                    }
+                }
+                Section("DESCRIPTION") {
+                    Text("Language: \(viewModel.repository.language ?? "n/a")")
+                        .textFontModifier(size: 20, weight: .semibold)
+                    Text(
+                        "Details: \(viewModel.repository.description ?? "n/a")"
+                    )
+                    .font(.title2)
+                }
+
+                Section("DATES") {
+                    Text(
```

Comment:
To stay consistent design wise, use HStack here as well. Left "Created at", spacer, right value.